### PR TITLE
Cluster Nodes Benutzungsrate

### PR DESCRIPTION
**Pull request by syjabri**

### CODE REVIEW

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Nutzungsrate der Cluster Nodes
+body_classes: title-center title-h1h2
+published: false
+---
+
+Während wir das Cluster geprüft haben ist uns eine ungewöhnliche Memory-Auslastung aufgefallen.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Bei der Überprüfung des Clusters ist uns eine ungewöhnlich hohe Speicher-Auslastung aufgefallen.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Es scheint so als ob ein Node komplett ausgelastet ist, während der zweite nur wenig Last zeigt.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Wir sehen, dass ein Node komplett ausgelastet ist, während der andere nur wenig Last zeigt.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Dies passiert oft, wenn ein Cluster mit weniger als drei Nodes ein Update an den Nodes durchführt. 
+Hier einmal grob erklärt, was passiert.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Hier erklären wir einmal, wie es dazu kommt.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## Verbinden
+
+Als erstes müssen wir wissen wie viele Nodes es gibt und wie die Auslastung ist.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Als erstes schauen wir, wieviele Nodes im Cluster sind, und wie ihre Auslastung ist.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+![Step 1](get_top_node_1.png)
+
+`Kubectl top node` zeigt die aktuelle Node Auslastung. Im Beispiel haben wie zwei Running(läuft ) Nodes.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Der Befehl `kubectl top node` zeigt die aktuelle Node Auslastung. Im Beispiel haben wie zwei Running (laufende) Nodes. 
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## Verbinden
+
+Als erstes müssen wir wissen wie viele Nodes es gibt und wie die Auslastung ist.
+![Step 1](get_top_node_1.png)
+
+`Kubectl top node` zeigt die aktuelle Node Auslastung. Im Beispiel haben wie zwei Running(läuft ) Nodes. 
+
+Erst wird der Node "cordoned" - also deaktiviert.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Erst wird der Node "cordoned" - also deaktiviert, damit keine neuen Pods auf dem Node gestartet werden.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+![Step 2](get_node_2.png)
+
+Dann wird der Node "drained" - also komplett leer gemacht und Pods auf alle anderen Nodes verteilt.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Dann wird der Node "drained" - also komplett leer gemacht und die bisher auf dem gedrainten Node laufenden Pods auf alle anderen Nodes des Clusters verteilt.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+Bei nur zwei Nodes wird also alles immer auf genau den einen anderen Node platziert.
+![Step 3](top_node_3.png)
+
+
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Wenn der zweite Node nach dem Update wieder läuft, werden die Pods _nicht_ automatisch auf beide Nodes verteilt. Dadurch kommt es zu dem eingangs beobachteten Ungleichgewicht.

```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+## Hier ein paar Tipps:
+Wir empfehlen mindestens drei Nodes zu verwenden, da so auch bei Updates Last besser verteilt werden kann.
+Im Webterminal gibt es ein Tool popeye, welches Hinweise auf Best Practices gibt: https://docs.imke.cloud/de/guides/webterminal.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Im Webterminal gibt es das Tool `popeye`, welches Hinweise auf Best Practices gibt: https://docs.imke.cloud/de/guides/webterminal.
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+---
+title: Nutzungsrate der Cluster Nodes
+body_classes: title-center title-h1h2
+published: false
+---
+
+Während wir das Cluster geprüft haben ist uns eine ungewöhnliche Memory-Auslastung aufgefallen.
+Es scheint so als ob ein Node komplett ausgelastet ist, während der zweite nur wenig Last zeigt.
+Dies passiert oft, wenn ein Cluster mit weniger als drei Nodes ein Update an den Nodes durchführt.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Dies passiert oft, wenn ein Cluster mit weniger als drei Nodes ein Upgrade an den Nodes durchführt. 
```

File: 02.Guides/08.CustomerSupport/default.de.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+![Step 2](get_node_2.png)
+
+Dann wird der Node "drained" - also komplett leer gemacht und Pods auf alle anderen Nodes verteilt.
+Bei nur zwei Nodes wird also alles immer auf genau den einen anderen Node platziert.
+![Step 3](top_node_3.png)
+
+
+## Hier ein paar Tipps:
+Wir empfehlen mindestens drei Nodes zu verwenden, da so auch bei Updates Last besser verteilt werden kann.
```
Comment (oliof) [inline 02.Guides/08.CustomerSupport/default.de.md]:
```suggestion
Wir empfehlen mindestens drei Nodes zu verwenden, da so auch bei Upgrades Last besser verteilt werden kann.
```

**Comment by chrisz100** [comment]
I'd suggest to rename the folder from CustomerSupport as this will be the sanity url grav will be publishing this at. Making this match the title is the current way of doing this

File: 02.Guides/08.NutzungsrateClusterNodes/default.de.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
+---
+title: Nutzungsrate der Cluster Nodes
+body_classes: title-center title-h1h2
+published: false
+---
+
+Bei der Überprüfung des Clusters ist uns eine ungewöhnlich hohe Speicher-Auslastung aufgefallen.
+Wir sehen, dass ein Node komplett ausgelastet ist, während der andere nur wenig Last zeigt.
+Dies passiert oft, wenn ein Cluster mit weniger als drei Nodes ein Upgrade an den Nodes durchführt.
```
Comment (oliof) [inline 02.Guides/08.NutzungsrateClusterNodes/default.de.md]:
```suggestion
Dies passiert oft, wenn bei einem  Cluster mit weniger als drei Nodes ein Upgrade an den Nodes durchgeführt wird. 
```

File: 02.Guides/08.NutzungsrateClusterNodes/default.de.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+Dann wird der Node "drained" - also komplett leer gemacht und die bisher auf dem gedrainten Node laufenden Pods auf alle anderen Nodes des Clusters verteilt.
+Bei nur zwei Nodes wird also alles immer auf genau den einen anderen Node platziert.
+![Step 3](top_node_3.png)
+
+Wenn der zweite Node nach dem Update wieder läuft, werden die Pods _nicht_ automatisch auf beide Nodes verteilt. Dadurch kommt es zu dem eingangs beobachteten Ungleichgewicht.
+
+## Hier ein paar Tipps:
+Wir empfehlen mindestens drei Nodes zu verwenden, da so auch bei Upgrades Last besser verteilt werden kann.
+Im Webterminal gibt es das Tool `popeye`, welches Hinweise auf Best Practices gibt: https://docs.imke.cloud/de/guides/webterminal.
```
Comment (oliof) [inline 02.Guides/08.NutzungsrateClusterNodes/default.de.md]:
```suggestion
Im Webterminal gibt es das Tool `popeye`, welches den Cluster analysiert und Verbesserungs-Vorschläge auf Grundlage von  Best Practices macht: https://docs.imke.cloud/de/guides/webterminal.
```

**Pull request merged**